code/game/machinery/doors/airlock.dm

```diff
@@ -739,6 +739,8 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
 			attack_ai(user)
 
 /obj/machinery/door/airlock/CanPass(atom/movable/mover, turf/target, height=0)
+	if(istype(mover) && mover.checkpass(PASSDOOR) && !locked) // We really dont want mice/drones to get shocked on a door they're passing through
+		return TRUE
```

Comment:
```suggestion
	if(istype(mover) && mover.checkpass(PASSDOOR)) // We really dont want mice/drones to get shocked on a door they're passing through
		return !locked // Disallow if door is bolted
```

Reply:
Same here. Before it would electrocute the mover if it was locked, now it doesn't. This could be correct or wrong. Just be sure to keep the change in mind